awx/main/models/ha.py

```diff
@@ -59,7 +59,7 @@ class Instance(HasPolicyEditsMixin, BaseModel):
         null=True,
         editable=False,
     )
-    version = models.CharField(max_length=24, blank=True)
+    version = models.CharField(max_length=120, blank=True)
```

Comment:
I wonder if having a different format for ansible-runner version output would break other versions of AWX. Maybe it worth changing the approach to use `pkg_resource` or a regex or doing a replace to find the version. I did some research about how would be the performance of each method and the `pkg_resource` version is the quickest and IMO more reliable one:

```python
In [16]: %timeit import subprocess; subprocess.check_output(['ansible-runner', '--version'], encoding='utf-8').replace('ansible-runner v', '').strip()                       
258 ms ± 4.62 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)

In [17]: %timeit import re; import subprocess; regex = re.compile(r'((\d+\.){2}\d+)'); regex.search(subprocess.check_output(['ansible-runner', '--version'], encoding='utf-8'
    ...: )).group(0)                                                                                                                                                         
264 ms ± 4.94 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)

In [18]: %timeit import pkg_resources; pkg_resources.get_distribution("ansible_runner").version                                                                              
276 µs ± 9.43 µs per loop (mean ± std. dev. of 7 runs, 1000 loops each)
```

I am not sure if we could access the same python as the runner is installed but if we can that would be the best option. If the previous one is not feasible, then I would go with the regex approach since we don't need to update the replace string if runner changes the version string again in the future.

Reply:
> would break other versions of AWX..

I'm not following - this change would go in the current version of awx and any future versions. Since we [pin the version of ansible-runner used by awx](https://github.com/ansible/awx/blob/devel/requirements/requirements.txt#L3), and old versions of runner have the shorter (non-breaking) version string, don't think that a backport is needed.